src/daemon.ts

```
@@ -54,12 +53,19 @@ class MqttWrapper implements MqttEndpoint {
       retain: true,
     });
   }
+
+  getBridgeAvailabilityTopicPath(): string {
+    return this.base_topic_ + '/bridge_availability';
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)